finance-admin-api/api/upload.go

```diff
@@ -96,18 +93,21 @@ func cleanString(s string) string {
 func (s *Server) upload(w http.ResponseWriter, r *http.Request) error {
 	ctx := r.Context()
 
+	var err error
 	var upload shared.Upload
 	defer r.Body.Close()
 
-	if err := json.NewDecoder(r.Body).Decode(&upload); err != nil {
+	if err = json.NewDecoder(r.Body).Decode(&upload); err != nil {
 		return err
 	}
 
 	useStrictHeaderComparison := upload.ReportUploadType != shared.ReportTypeUploadPaymentsSupervisionCheque
 
-	err := validateCSVHeaders(upload.File, upload.ReportUploadType, useStrictHeaderComparison)
-	if err != nil {
-		return err
+	if upload.ReportUploadType != shared.ReportTypeUploadDirectDebitsCollections {
```

Comment:
Why do we not validate headers for this upload?

Reply:
This upload does not have any headers so we cant validate anything unfortunately here